conf/scylla.yaml

```diff
@@ -837,3 +837,6 @@ maintenance_socket: ignore
 # Note that creating keyspaces with tablets enabled or disabled is irreversible.
 # The `tablets` option cannot be changed using `ALTER KEYSPACE`.
 enable_tablets: true
+
+# Enforce RF-rack-valid keyspaces.
+rf_rack_valid_keyspaces: false
```

Comment:
Even if `rf_rack_valid_keyspaces: false` there should be a warning about using RF!=#Racks, letting users know this option will be deprecated in the future.
Warnings should exist in:
- CQLSh, when creating, altering a KS
- Traces

Reply:
https://github.com/scylladb/scylladb/issues/23330